### PR TITLE
docs(developer-guide): update the compiling guide of stonedb8.0 on centos7.x #1817

### LINKED ISSUE #1817: docs(developer-guide): update the compiling guide of StoneDB 8.0 for CentOS 7.x
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Description

update compiling guide of stonedb8.0 on CentOS7.x 

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by Xinqiangxu**
## Summary about this PR

Issue Number: close #1817 

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [x] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: Docs/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Ensure that the tools and third-party libraries used in your environment meet th
 
 :::info
 Permission issues may occur when executing the following commands. You are advised to run the commands with the administrator permission
+Note: All commands in this article are run under root privileges by default
```
Comment (Nliver) [inline Docs/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md]: For Notes, you can use the `tip` tag. See this link for how different tags can be rendered: https://docusaurus.io/docs/markdown-features/admonitions

File: website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ sidebar_position: 5.121
  * boost 1.77
 
 :::info 
-以下命令的执行可能会遇到权限问题，建议在管理员权限下运行
+以下命令的执行可能会遇到权限问题，建议在root权限下运行
+注：本文中所有命令默认是在root权限下运行
```
Comment (Nliver) [inline website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md]: For Notes, you can use the `tip` tag. See this link for how different tags can be rendered: https://docusaurus.io/docs/markdown-features/admonitions

**Review by Nliver — CHANGES_REQUESTED** [review]

File: website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md (hunk 2 above)
Comment (Xinqiangxu) [reply website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-centos7/compile-using-centos7-for-80.md]: ok

**Review by Nliver — APPROVED** [review]
LGTM

**Review by hustjieke — CHANGES_REQUESTED** [review]
Please do git rebase, so many commits

**Pull request closed without merging**